src/endpoints/endpoints.spec.ts

```diff
@@ -0,0 +1,10 @@
+import {describe, expect, it} from 'vitest';
+import * as Endpoints from '.';
+
+describe('Endpoints', () => {
+  it.each(Object.values(Endpoints))(`%o has required properties`, (Endpoint) => {
```

Comment:
wat is %0 ?

Reply:
formatting van de each value https://vitest.dev/api/#test-each